src/Tribe/Tickets.php

```diff
@@ -2416,31 +2416,40 @@ public function maybe_redirect_to_attendees_registration_screen( $redirect = nul
 				return;
 			}
 
+			$is_paypal              = ! ! $redirect;
```

Comment:
Whoa, is this intentional? There's also a `if ( ! $is_paypal )` line below at 2426

Reply:
!!? and then a ! check later?

Reply:
Should I just `(bool)` cast it instead? the only two possible values I'm seeing are a `string` or `null`.

Reply:
I just did it.